ups/eupspkg.cfg.sh

```diff
@@ -17,11 +17,11 @@ install()
     case $(uname -s) in
         Linux*)
             ana_platform="Linux-x86_64"
-            conda_packages="conda_packages-linux-64.txt"
+            conda_packages="conda2_packages-linux-64.txt"
             ;;
         Darwin*)
             ana_platform="MacOSX-x86_64"
-            conda_packages="conda_packages-osx-64.txt"
+            conda_packages="conda2_packages-osx-64.txt"
```

Comment:
Are you sure this change is desirable? I think we considered doing this before but then backed out of it when we realized that miniconda is versioned with eups but lsstsw is not. So depending on which build you are recreating you may get a failure if your lsstsw is expecting different forms.